zk/txpool/metrics.go

```diff
@@ -41,16 +41,19 @@ func (m *Metrics) Update(pool *TxPool) {
 	currentPoolSize := uint64(pool.pending.Len() + pool.baseFee.Len() + pool.queued.Len())
 
 	out := m.lastPoolSize + m.txCounter - currentPoolSize
-	if out < 0 {
-		out = 0
-	}
```

Comment:
How come we removed this?

Reply:
My bad!  Although that does make me wonder about underflows but honestly for some metrics endpoint that shouldn't be a worry for us.